util/partition_integers

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# partition_integers -- Partitions integer range into even ranges
+##
+import sys
+
+M = int(sys.argv[1])
+N = int(sys.argv[2])
+
+k = float(M)/N
```

Comment:
Can we add a check for N here? E.g. N=0.
Also, if M=10 and N=20, the answer is
1 0 0
0 1 0
1 1 1
0 2 1
1 2 2
0 3 2
1 3 3
0 4 3
1 4 4
0 5 4
Is this answer correct?